x-pack/plugins/ml/public/maps/anomaly_source_field.ts

```diff
@@ -40,18 +40,56 @@ export const ANOMALY_SOURCE_FIELDS: Record<string, Record<string, string>> = {
     }),
     type: 'string',
   },
+  actual: {},
```

Comment:
nit: suggestion to add a comment to clarify what's the difference between `actual` and `actualDisplay` here or why this `actual` here doesn't need a label.

Reply:
Added in 2321384960d2018fc3aa3e54ddf0d4e5c95ae80a